modules/core/src/main/scala/org/scalasteward/core/edit/hooks/HookExecutor.scala

```diff
@@ -59,24 +59,36 @@ final class HookExecutor[F[_]](implicit
 }
 
 object HookExecutor {
-  val postUpdateHooks: List[PostUpdateHook] =
-    List(
+
+  private[this] val knownGitHubActions = List(
+    (GroupId("com.codecommit"), ArtifactId("sbt-github-actions")),
+    (GroupId("com.codecommit"), ArtifactId("sbt-spiewak")),
+    (GroupId("com.codecommit"), ArtifactId("sbt-spiewak-sonatype")),
+    (GroupId("com.codecommit"), ArtifactId("sbt-spiewak-bintray"))
+  )
+
+  val postUpdateHooks: List[PostUpdateHook] = {
+    val actionsActions = knownGitHubActions.map { case (gid, aid) =>
+      PostUpdateHook(
+        groupId = gid,
+        artifactId = aid,
+        command = Nel.of("sbt", "githubWorkflowGenerate"),
+        useSandbox = true,
+        commitMessage =
+          update => s"Regenerate workflow with sbt-github-actions ${update.nextVersion}",
```

Comment:
```suggestion
        commitMessage = _ => "Regenerate workflow with sbt-github-actions",
```
Including the version in the commit message could be confusing because `update.nextVersion` is the version of `aid` which is not necessarily the version of sbt-github-actions.

Reply:
Ah good point.